web3.md

```diff
@@ -852,7 +852,7 @@ Transaction Receipts
          </message>
          <account>
            <acctID> ACCTFROM </acctID>
-           <balance> BAL => BAL -Int (GLIMIT *Int GPRICE) </balance>
+           <balance> BAL => BAL -Int 2 *Int (GLIMIT *Int GPRICE) </balance>
```

Comment:
For some reason, the `from` account ends with too much gas being refunded (as if it is being refunded twice), and so far I haven't been able to find out where this is happening.

Reply:
@gtrepta it looks like `finalizeTx` is already refunding gas to the account (at the same time that it awards the miner gas.

So you probably don't need `#catchHaltTx _` to do a `#refund GAVAIL`

Reply:
I don't think that's it. `#refund` adds to the `<gas>` cell. It's used after a `#popCallStack` to preserve the remaining gas.